connect/src/com/telenor/connect/AbstractSdkProfile.java

```diff
@@ -166,6 +166,10 @@ protected void previewParameters(ParametersHolder parameters) {
                         && isEmpty(wellKnownConfig.getNetworkAuthenticationTargetUrls()))) {
                     parameters.add("login_hint", String.format("MCCMNC:%s", mccMnc));
         }
+
+        if (!ConnectSdk.isCellularDataNetworkConnected()) {
```

Comment:
I'm not sure what we are discussing here :)
I agreed with you in my 2nd comment.
I just ask you to put in the description that with your patch you also add Bluetooth and USB functionality to Forced HE.
And also I don't agree it's a bug as this is by design and it's quite well documented in the link in my 2nd comment. The reason these comments are not part of the repo, but are only in Confluence is that because this repo is public.